lib/auth/init.go

```diff
@@ -105,12 +101,11 @@ func Init(cfg InitConfig, seedConfig bool) (*AuthServer, *Identity, error) {
 		return nil, nil, trace.BadParameter("HostUUID: host UUID can not be empty")
 	}
 
-	lockService := local.NewLockService(cfg.Backend)
-	err := lockService.AcquireLock(cfg.DomainName, 60*time.Second)
+	err := cfg.Backend.AcquireLock(cfg.DomainName, 30*time.Second)
 	if err != nil {
 		return nil, nil, err
 	}
-	defer lockService.ReleaseLock(cfg.DomainName)
+	defer cfg.Backend.ReleaseLock(cfg.DomainName)
```

Comment:
Well... the goal is to remove locks completely. This PR reduces the number of lock-related code to an absolute minimum to discourage lock usage and simplify subsequent removal of them.